draft-ietf-tls-dtls-connection-id.md

```diff
@@ -134,34 +133,48 @@ indicates that the server will send with the client's CID but does not
 wish the client to include a CID (or again, alternately, to use a
 zero-length CID).
 
-When a session is resumed, the "connection_id" extension is
-negotiated afresh, not retained from previous connections in
-the session.
-
-Because each party sends the value in the "connection_id" extension that it wants to 
-receive as a connection identifier in encrypted records, it is possible
+Because each party sends the value in the "connection_id" extension it wants to 
+receive as a CID in encrypted records, it is possible
 for an endpoint to use a globally constant length for such connection
 identifiers.  This can in turn ease parsing and connection lookup,
-
 for example by having the length in question be a compile-time constant.
 Implementations, which want to use variable-length CIDs, are responsible
 for constructing the CID in such a way that its length can be determined
-on reception. Note that such implementations must still be able to send
-connection identifiers of different length to other parties.
-
-Note that it is not possible to parse the records without knowing how 
-long the CID is.
+on reception. Such implementations must still be able to send
+CIDs of different length to other parties. Is not possible to parse 
+the records without knowing how long the CID is.
```

Comment:
I'm not sure, why "Is not possible to parse the records without knowing how long the CID is." is kept. I tried to address my concerns in #27 . But this may be considered as independent from this PR.

Reply:
Maybe it would be better to say that there is no CID length information included in the record itself. I think this would clarify the issue. Do you agree?

Reply:
Yes, that sounds good.